Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Scrollbar.md

![CreateWithProperties](./media/scrollbar_properties.PNG)

> [!NOTE]

> > ScrollBar is deprecated in API8 (Tizen 6.0) and will be removed in API10.

Reviewer: Global

@surbhig12 Could you please re-write this sentence also? I think it would be better to remove the parentheses in the sentence and

Reviewer: @surbhig12 Would you please check this comment and the updated?

Reviewer: FYI to @surbhig12 
At https://github.com/Samsung/tizen-docs/pull/1272/files which is still on reviewing, 
API8 & Tizen 6.0 is written in below format.

FlexContainer class is deprecated since Tizen 6.0 and will be supported for the subsequent two releases only.

I'd like to ask your opinion, related with this line.

Reviewer 2: @mijin85cho,

I would suggest the following:

FlexContainer class is deprecated since Tizen version 6.0 and will be removed after two releases.

Reviewer: @surbhig12 I'd like to add my  opinion to your suggestion. How about removing "version" after Tizen as it is not common use in Tizen docs? The sentence format would be like this:

\<Component name> is deprecated since Tizen 6.0 and will be removed after two releases.

ex) 
Popup is deprecated since Tizen 6.0 and will be removed after two releases.
ScrollBar is deprecated since Tizen 6.0 and will be removed after two releases.
Tab is deprecated since Tizen 6.0 and will be removed after two releases.
...

Reviewer 2: I agree. Thank you.

Reviewer 2: Please rephrase to:

ScrollBar is deprecated since Tizen 6.0 and will be removed after two releases.

> ## Add namespace using
> ScrollBar is defined in namespace Tizen.NUI.Components.

Reviewer 2: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement scrollbar, add namespace `using Tizen.NUI.Components` in your application:
```
